corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml

```diff
@@ -181,8 +181,16 @@
 
 - id: persistent_menu
   name: Show Persistent Menu
-  description: Show a persistent menu instead of breadcrumbs.
+  description: Show the persistent menu on the side of the web app.
   toggle: PERSISTENT_MENU_SETTING
   widget: bool
   default: false
   since: '2.54'
+
+- id: show_breadcrumbs
+  name: Show Breadcrumbs
+  description: Show the breadcrumbs on top of the web app.
+  toggle: PERSISTENT_MENU_SETTING
```

Comment:
Is this meant to be behind the persistent menu feature flag?

Reply:
Yes, I did not want to create yet another FF. And removing the breadcrumbs does not really make sense without showing the persistent menu

Reply:
Would a feature flag negate the need for an app setting?

Reply:
I think we don't really need an app setting for peristent menu but enabling ff should be enought to enable them for the domain. 

Although we might want fine grained control on breadcrumbs as product wants to have ability to have both breadcrumbs and persistent menu enabled for a project to facilititate smooth user transition from breadcrumbs to peristent Menu.